src/rpc/rpcevo.cpp

```diff
@@ -588,7 +588,7 @@ void protx_revoke_help()
 
 UniValue protx_revoke(const JSONRPCRequest& request)
 {
-    if (request.fHelp || (request.params.size() != 2 && request.params.size() != 3)) {
+    if (request.fHelp || (request.params.size() != 3 && request.params.size() != 4)) {
```

Comment:
While at it, let's also fix help message (line 573) - should be
```
"protx revoke \"proTxHash\" \"operatorKey\" ( \"reason\" )\n"
```

Reply:
done